packages/manager/modules/order/src/order.constant.ts

```diff
@@ -59,12 +59,15 @@ export const getVcdProductSettings = ({
 export const getHYCUProductSettings = ({
   planCode,
   region,
+  serviceName = '',
 }: {
   planCode: string;
   region: string;
+  serviceName?: string;
 }) =>
```

Comment:
```suggestion
}: HYCUOrder) =>
```

It's the same type ?

Reply:
HYCU types are not available in order module